app/src/main/kotlin/com/wire/android/ui/authentication/create/email/CreateAccountEmailScreen.kt

```diff
@@ -254,7 +267,8 @@ private fun EmailFooter(state: CreateAccountEmailViewState, onLoginPressed: () -
         state = if (state.continueEnabled) WireButtonState.Default else WireButtonState.Disabled,
         modifier = Modifier
             .fillMaxWidth()
-            .padding(MaterialTheme.wireDimensions.spacing16x),
+            .padding(MaterialTheme.wireDimensions.spacing16x)
+            .imePadding()
```

Comment:
It makes the "continue" button disappear when the keyboard is visible on Android 9: 
![imepadding](https://github.com/wireapp/wire-android-reloaded/assets/30429749/f802c9b9-cda1-423b-89c6-37577eb693cc)